paper-radio-group.html

```diff
@@ -71,116 +69,74 @@
 
     properties: {
       /**
-       * Fired when the selected element changes to user interaction.
-       *
-       * @event paper-radio-group-changed
+       * Overriden from Polymer.IronSelectableBehavior
```

Comment:
I didn't want to copy the comment from the behavior; does this kind of make sense?